crates/driver/src/tests/cases/mod.rs

```diff
@@ -41,3 +41,6 @@ pub const DEFAULT_SCORE_MAX: u128 = 500000000000000000000000000000u128;
 
 /// The default solver fee for limit orders.
 pub const DEFAULT_SOLVER_FEE: u128 = 100u128;
+
+/// The default maximum value to be payout out to solver per solution
+pub const DEFAULT_SCORE_CAP: u128 = 10000000000000000u128;
```

Comment:
@fhenneke This comment I had to add make it obvious that `score_cap` is a bad name for this. Btw, `score_cap` is used in [this function](https://github.com/cowprotocol/services/blob/main/crates/solver/src/settlement_rater.rs#L405).

Do we have a more descriptive alternatives to name this variable? `reward_cap`, `payout_cap` or something else?

Right now, when I look into autopilots `score_cap`, based on it's name it looks like it is some kind of maximum score `driver` is allowed to submit to autopilot...

Reply:
Indeed `reward_cap` is a more descriptive name. The value puts a limit on how much we pay on top of gas reimbursements, i.e. the reward.
The number does not on its own give a bound on the total payment. The payment is bounded by `reward_cap + gas_costs`.